### PR TITLE
Refactor cert payload

### PR DESCRIPTION
**Pull request by m-bo-one**
Just for diff change. No merge.

### CODE REVIEW

File: certificate.proto [hunk 1]
```diff
@@ -23,9 +23,21 @@ message CertificateMethod {
     }
 }
 
+message RSAPayload {
+    int32 key_size = 1;
+    int32 public_exponent = 2;
+}
+
 message NewCertificatePayload {
```
Comment (apadenko) [inline certificate.proto]: Let's rename Certificate to PublicKey. 
NewPublicKeyPayload, 
RevokePublicKeyPayload .... 
@eugene-babichenko agree with that ?

File: certificate.proto [hunk 2]
```diff
@@ -35,6 +47,18 @@ message NewCertificatePayload {
     // for certificate signing. As far as public key will be
     // a part of transaction payload
     string cert_signer_public_key = 4;
+    PubKeyType public_key_type = 5;
+    // For future ACME support(Now only Personal)
+    EntityType entity_owner_type = 6;
+    RSAPayload rsa_parameters = 7;
+    // hash of PEM encoded public key of certificate
+    string entity_hash = 8;
```
Comment (apadenko) [inline certificate.proto]: we also need entity_hash_signature in creation payload. 
@eugene-babichenko  agree with that ?

Comment (eugene-babichenko) [reply certificate.proto]: This is `crt_sig`.

File: certificate.proto (hunk 1 above)
Comment (eugene-babichenko) [reply certificate.proto]: Yes.

**Review by eugene-babichenko — APPROVED** [review]

File: certificate.proto (hunk 2 above)
Comment (eugene-babichenko) [reply certificate.proto]: Rename `crt_sig` -> `entity_hash_signature`
Remove `rem_signature`

**Review by eugene-babichenko — CHANGES_REQUESTED** [review]

File: certificate.proto [hunk 3]
```diff
@@ -35,6 +47,18 @@ message NewCertificatePayload {
     // for certificate signing. As far as public key will be
     // a part of transaction payload
     string cert_signer_public_key = 4;
+    PubKeyType public_key_type = 5;
+    // For future ACME support(Now only Personal)
+    EntityType entity_owner_type = 6;
+    RSAPayload rsa_parameters = 7;
```
Comment (eugene-babichenko) [inline certificate.proto]: Remove (present in PEM)

**Review by eugene-babichenko — APPROVED** [review]

**Pull request merged**